rasa/core/http_interpreter.py

```diff
@@ -50,6 +50,7 @@ async def _rasa_http_parse(
             structlogger.error(
                 "http.parse.text",
                 text=copy.deepcopy(text),
+                event_info="No rasa NLU server specified!",
```

Comment:
We should also document this in some code conventions in Notion or maybe the Readme so that in the future everyone is aware that this key is for human readable text?